crates/sui-indexer/src/store/pg_indexer_store.rs

```diff
@@ -539,6 +540,25 @@ impl<T: R2D2Connection + 'static> PgIndexerStore<T> {
         if checkpoints.is_empty() {
             return Ok(());
         }
+        // If the first checkpoint has sequence number 0, we need to persist the digest as
+        // chain identifier.
+        let first_checkpoint = checkpoints.first().unwrap();
```

Comment:
If there's no first checkpoint, there'd be a lot of issues, but maybe it's better to handle this error properly rather than panic.

Reply:
Just noticed this -- there is an `is_empty` check just above here so that's what's guarding the `unwrap`, but I've refactored this to use `let Some(...)` instead.

Reply:
Ah yes, did not read above the change.